packages/rosetta/configuration/run_tests.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+set -e
+
+wait_for_rosetta() {
+  timeout 90 sh -c 'until nc -z $0 $1; do sleep 1; done' rosetta 8080
```

Comment:
FYI, timeout is not a standard UNIX command, but this is okay if this script is only run under Docker.

Reply:
Rewrote as a bash while loop..